src/views/menu.dot

```diff
@@ -0,0 +1,43 @@
+<div class="menu__overlay" {{= click(this.close, this) }}></div>
+
+<div class="menu__button" {{= click(this.open, this) }}>
+  <div class="menu__button__line"></div>
+  <div class="menu__button__line"></div>
+  <div class="menu__button__line"></div>
+</div>
+
+<div class="menu">
+  <div class="menu__panel">
+  <div class="menu__panel__top">
+    <i class="icon-x menu__panel__top__close" {{= click(this.close, this) }}></i>
+  </div>
+    <h2 class="menu__panel__section_title">
+      <i class="menu__panel__section__icon icon-map"></i>
+      Qwant Maps
+    </h2>
+    <div class="menu__panel__section">
+      <button class="menu__panel__action" {{= click(this.toggleDirection, this) }}>{{= _('Itinerary' , 'menu') }}</button>
+      {{? this.isDirectionActive }}
+        <button class="menu__panel__action" {{= click(this.toggleFavorite, this) }}>{{= _('Favorites', 'menu') }}</button>
+      {{?}}
+    </div>
+
+    {{~ this.menuItems:menuItem }}
+      <h2 class="menu__panel__section_title">
+        <i class="menu__panel__section__icon icon-{{= menuItem.icon }}"></i>
+        {{= _(menuItem.sectionName) }}
+      </h2>
+      <div class="menu__panel__section">
```

Comment:
What about the "Contact us" menu ?  
The section name should be a link itself.

Reply:
forgot to manage that case